pkg/server/event_handler.go

```diff
@@ -230,6 +230,7 @@ func (h *EventHandler[T, P]) HTTPHandler() http.Handler {
 			Data:       m.Message.Data, // Notification payload.
 			Attributes: m.Message.Attributes,
 		}
+		//nolint:sloglint
```

Comment:
Why this is needed?

Reply:
Linters complain. it doesn't allow to use snakecase. But `bucketID` is snakecase